smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/middlewares/OperationInputBodyMiddleware.kt

```diff
@@ -21,7 +21,9 @@ import software.amazon.smithy.swift.codegen.SwiftWriter
 import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
```

Comment:
Depending on the request wire protocol, either pass in the utility method defined for JSON, or pass in the newly-added code-generated `marshal` static function variable of the streaming shape.